katharsis-core/src/main/java/io/katharsis/core/internal/resource/AnnotationResourceInformationBuilder.java

```diff
@@ -110,7 +110,8 @@ public String getResourceType(Class<?> resourceClass) {
 			ResourceFieldType resourceFieldType = AnnotatedResourceField.getResourceFieldType(annotations);
 			String oppositeResourceType = resourceFieldType == ResourceFieldType.RELATIONSHIP ? getResourceType(field.getGenericType(), context) : null;
 			AnnotatedResourceField resourceField = new AnnotatedResourceField(jsonName, underlyingName, field.getType(), field.getGenericType(), oppositeResourceType, annotations);
-			if (Modifier.isTransient(field.getModifiers()) || Modifier.isStatic(field.getModifiers())) {
+			if (Modifier.isTransient(field.getModifiers()) || Modifier.isStatic(field.getModifiers())
+                    || resourceField.isAnnotationPresent(JsonIgnore.class)) {
 				fieldWrappers.add(new ResourceFieldWrapper(resourceField, true));
```

Comment:
the only difference between these 2 branches is that lone boolean. This whole `if` could be swapped out